examples/webgpu_loader_gltf.html

```diff
@@ -79,7 +80,7 @@
 				renderer = new THREE.WebGPURenderer( { antialias: true } );
 				renderer.setPixelRatio( window.devicePixelRatio );
 				renderer.setSize( window.innerWidth, window.innerHeight );
-				renderer.toneMapping = THREE.ACESFilmicToneMapping;
+				// renderer.toneMapping = THREE.ACESFilmicToneMapping;
```

Comment:
Important note – omission of tone mapping is an error in the PlayCanvas implementation, and others I've seen so far. We do still need tone mapping, for SDR as well as HDR, as we are still forming an image for a display. The display is of course brighter than before, and will require adaptations to the tone mappers.

Reply:
I'm aware, good point! – added to the DRAFT reasons above.

I did some testing on this as well: Mac displays are supposed to be ~3x brighter in HDR mode (1600 nits instead of 500 for SDR content), from my limited testing the practical range ("how far can I crank emissiveIntensity or environmentIntensity") seems to be ~20-30x brighter though. So there might already be tonemapping on the hardware side as well.

This image also behaves kind of suspicious: https://www.instagram.com/p/C_Byx6jRfuX/
When adjusting screen brightness while it's open, there are noticeable "steps" where the display goes into different modes (for a lack of better term: "tonemapping modes") with somewhat different relative feel.

Reply:
Oh, ouch! I was aware that adjusting screen brightness changed the available “HDR” headroom, and that this would be a problem with the current WebGPU HDR implementation, but I hadn't seen a clear demo of the issue like that. Really helpful illustration — thanks.